crates/breez-sdk/common/src/input/models.rs

```diff
@@ -158,6 +158,119 @@ pub enum InputType {
     Bip21(Bip21Details),
     Bolt12InvoiceRequest(Bolt12InvoiceRequestDetails),
     LnurlWithdraw(LnurlWithdrawRequestDetails),
+    SparkAddress(SparkAddressDetails),
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SparkAddressDetails {
+    pub encoded_address: String,
+    pub decoded_address: SparkAddress,
+    pub source: PaymentRequestSource,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
+pub struct SparkAddress {
+    pub identity_public_key: String,
+    pub network: BitcoinNetwork,
```

Comment:
This should be a spark network, not a bitcoin network? In fact they are the same, so maybe it doesn't really matter.